Tests/SwiftParserTest/translated/RecoveryTests.swift

```diff
@@ -1962,14 +1962,21 @@ final class RecoveryTests: XCTestCase {
       """
       // <rdar://problem/22387625> QoI: Common errors: 'let x= 5' and 'let x =5' could use Fix-its
       func r22387625() {
-        let _= 5
-        let _ =5
+        let _1️⃣= 4
+        let _ =2️⃣5
       }
       """,
       diagnostics: [
-        // TODO: Old parser expected error on line 3: '=' must have consistent whitespace on both sides, Fix-It replacements: 8 - 8 = ' '
-        // TODO: Old parser expected error on line 4: '=' must have consistent whitespace on both sides, Fix-It replacements: 10 - 10 = ' '
-      ]
+        DiagnosticSpec(locationMarker: "1️⃣", message: "'=' must have consistent whitespace on both sides"),
+        DiagnosticSpec(locationMarker: "2️⃣", message: "'=' must have consistent whitespace on both sides"),
+      ],
+      fixedSource: """
+        // <rdar://problem/22387625> QoI: Common errors: 'let x= 5' and 'let x =5' could use Fix-its
```

Comment:
If you want, you can extract this comment from the string literal. It’s just an artifact of the translation that it ended up in the literal instead of being a real comment.